pydeconz/models/sensor/air_quality.py

```diff
@@ -78,6 +90,21 @@ def pm_2_5(self) -> int | None:
         """Air quality PM2.5 (µg/m³)."""
         return self.raw["state"].get("pm2_5")
 
+    @property
+    def supports_air_quality(self) -> bool:
+        """Support Air quality reporting."""
+        return "airquality" in self.raw["state"]
+
+    @property
+    def supports_air_quality_co2(self) -> bool:
+        """Support Air quality CO2 reporting."""
+        return "airqualityco2" in self.raw["state"]
+
+    @property
+    def supports_air_quality_formaldehyd(self) -> bool:
```

Comment:
```suggestion
    def supports_air_quality_formaldehyde(self) -> bool:
```
At the very least it can be properly spelled on pydeconz side :)